commands/bridge_configure.go

```diff
@@ -98,4 +113,10 @@ var bridgeConfigureCmd = &cobra.Command{
 
```

Comment:
Ideally, the `bridge configure` command would have a `Long` description that explain shortly the process (flags or automated process, token or login, ..) and which flags are valid for which bridge. Kinda what you wrote to present the PR.

Reply:
Done